dipy/align/tests/test_vector_fields.py

```diff
@@ -919,21 +952,19 @@ def test_compose_vector_fields_2d():
     input_grid2world = gt_affine
     target_grid2world = gt_affine
 
-    disp1, assign1 = vfu.create_random_displacement_2d(np.array(input_shape,
-                                                       dtype=np.int32),
-                                                       input_grid2world,
-                                                       np.array(tgt_sh,
-                                                       dtype=np.int32),
-                                                       target_grid2world)
+    disp1, assign1 = vfu.create_random_displacement_2d(
+        np.array(input_shape, dtype=np.int32), 
```

Comment:
We have some trailing white spaces in these lines too, this is what `flake8` reports on `test_vector_fields.py`:
```
test_vector_fields.py:956:47: W291 trailing whitespace
test_vector_fields.py:957:26: W291 trailing whitespace
test_vector_fields.py:958:42: W291 trailing whitespace
test_vector_fields.py:964:47: W291 trailing whitespace
test_vector_fields.py:965:26: W291 trailing whitespace
test_vector_fields.py:966:42: W291 trailing whitespace
test_vector_fields.py:1091:47: W291 trailing whitespace
test_vector_fields.py:1092:60: W291 trailing whitespace
```